### ISSUE TITLE
amplify configure project command does not properly update frontendHandler.javascript path

### ISSUE THREAD

**Issue by chrisl777** [aws-amplify/amplify-cli#296]
**Describe the bug**
I am on a development team where we want to have multiple developers running the same amplify project. Another developer created a basic Amplify project for a React app. When I run the `amplify configure project` command on my machine, and then run an `amplify push`, I get the following error: 

> Cannot find module '/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/amplify-frontend-javascript'

**To Reproduce**
1. Check out repo of an Amplify project. 
2. cd into the project directory. 
3. `npm i` 
4. Run `amplify configure project` to update the project settings for your machine (this updates amplify/.config/project-config.json for one thing). 
5. Run `amplify push`.

**Expected behavior**
The `amplify push` should complete as normal. 

**Desktop (please complete the following information):**
-macOS 

**Additional context**
Note the project-config.json file below after running `amplify configure project`, my providers.awscloudformation is correct, but frontendHandler.javascript is not. Perhaps the issue may stem from the fact that I'm using nvm to manage my node installs. 

```
{
    "projectName": "myprojectname",
    "projectPath": "/Users/myname/Documents/projectfolder/project",
    "defaultEditor": "code",
    "frontendHandler": {
        "javascript": "/usr/local/lib/node_modules/@aws-amplify/cli/node_modules/amplify-frontend-javascript"
    },
    "javascript": {
        "framework": "react",
        "config": {
            "SourceDir": "src",
            "DistributionDir": "build",
            "BuildCommand": "npm run-script build",
            "StartCommand": "npm run-script start"
        }
    },
    "providers": {
        "awscloudformation": "/Users/myname/.nvm/versions/node/v8.12.0/lib/node_modules/@aws-amplify/cli/node_modules/amplify-provider-awscloudformation"
    }
}
```

**Comment by UnleashedMind** [comment]
Team collaboration workflow has not been fully implemented yet, what you are doing is not going to work. 
We are working on implement it. 
I'm marking this as feature request.

**Comment by kaustavghosh06** [comment]
@chrisl777 were able to get your app running by manually changing the path? We’ll fix this issue as a part of the ‘amplify configure project’ command

**Comment by chrisl777** [comment]
@kaustavghosh06 Yes, thanks, I was able to get the app running by manually changing the path. Thanks for putting that into your "to fix" list!

**Comment by kaustavghosh06** [comment]
The fix for this is merged in the latest version of the CLI -> 0.1.28

**Comment by ayushme001** [comment]
try this
$export PATH="/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Users/YOURUSERNAME/.npm-global/bin"

HOPE it might work!! 🙂
